static/src/javascripts/projects/common/modules/commercial/creatives/expandable-v2.js

```diff
@@ -77,29 +131,47 @@ define([
                 showPlus: (this.params.showMoreType === 'plus-only' || this.params.showMoreType === 'plus-and-arrow') ?
                     '<button class="ad-exp__close-button ad-exp__open">' + svgs('closeCentralIcon') + '</button>' : ''
             },
-            $expandablev2 = $.create(template(expandableV2Tpl, _.merge(this.params, showmoreArrow, showmorePlus, videoDesktop)));
+            scrollingbg = {
+                scrollbg: (this.params.backgroundImagePType !== '' || this.params.backgroundImagePType !== 'none') ?
+                    '<div class="ad-exp--expand-scrolling-bg" style="background-image: url(' + this.params.backgroundImageP + '); background-position: ' + this.params.backgroundImagePPosition + ' 50%;"></div>' : ''
```

Comment:
I think that this:
$.create('<div>')
  .addClass('foo')
  .css({
    color: 'red',
    'background-color': 'white'
  })

is more readable that your one liner. But I am not sure if it is worth to update it everywhere.